corehq/apps/export/forms.py

```diff
@@ -919,30 +921,39 @@ def get_filters(self, can_access_all_locations, accessible_location_ids, group_i
         :param location_ids:
         :param date_range: A DatePeriod or DateSpan
         """
-        form_filters = []
-        if can_access_all_locations:
-            form_filters += [_f for _f in [
-                self._get_group_filter(group_ids),
-                self._get_user_type_filter(user_types),
-            ] if _f]
-
-        form_filters += [_f for _f in [
-            self._get_users_filter(user_ids),
-            self._get_locations_filter(location_ids)
-        ] if _f]
-
-        form_filters = flatten_non_iterable_list(form_filters)
-        if form_filters:
-            form_filters = [OR(*form_filters)]
-        else:
-            form_filters = []
+
+        form_filters = list(filter(None,
+                                   [self._create_user_filter(user_types, user_ids, group_ids, location_ids)]))
         date_filter = self._get_datespan_filter(date_range)
         if date_filter:
             form_filters.append(date_filter)
```

Comment:
[not blocking] You can move this in the same list as above, as in:
```python
form_filters = list(filter(None, [
    self._create_user_filter(user_types, user_ids, group_ids, location_ids),
    self._get_datespan_filter(date_range),
]))
```
and if you like, also the scope filter:
```python
self._scope_filter(...) if not can_access_all_locations else None
```